certified-connectors/secplugs/apiDefinition.swagger.json

```diff
@@ -1,12 +1,12 @@
 {
   "swagger": "2.0",
   "info": {
-    "title": "Secplugs Scan",
-    "description": "Secplugs Scan for scanning Emails content, files contents and Urls, Secplugs scan is a platform which creates hybrid security platform using various security provider",
+    "title": "Scancloud APIs",
+    "description": "Scancloud APIs for scanning Emails content, files contents and Urls, Scancloud APIs is a platform which creates hybrid security platform using various security provider",
     "version": "1.0",
     "contact": {
-      "name": "Secplugs",
-      "url": "https://secplugs.com/",
+      "name": "Scancloud",
+      "url": "Website	https://scancloud.io/",
```

Comment:
@Ajaatsecplugs the URL property should contain only contact URL, please remove the word Website and whitespaces from the string.